packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature

```diff
@@ -0,0 +1,93 @@
+@owner:andy @author:andy
+Feature: Docusign Form
+
+  Background: A user has submitted their information in Step 1 and Step 2
+    Given a user is on the Docusign Form
+
+  @minutia
+  Scenario: Docusign URL is set in iframe on page load
+    When the Docusign Form loads
+    And the Docusign URL has been fetched
+    Then the URL set is set on the Docusign iframe
+    And the iframe loads based on the URL
+
+  @minutia
+  Scenario: Web-page fails when Docusign URL cannot be fetched
+    When the Docusign Form loads
+    And the Docusign URL cannot be fetched
+    Then the URL is not set on the Docusign iframe
+    And the iframe fails to load
+
+  @minutia
+  Scenario Outline: Participant not based in the US working with US brand selects participantType
+    When the page loads
```

Comment:
This spec should have the state of the participant and the brand as givens before the when